modules/react/common/stories/Responsive.stories.mdx

```diff
@@ -0,0 +1,333 @@
+import {Meta, ArgsTable} from '@storybook/addon-docs';
+import {Box} from '@workday/canvas-kit-react/layout';
+//
```

Comment:
is this supposed to be here ?

Reply:
The comment slashes no. I will update that.